src/pages/EmergencyViewPage/EmergencyViewRoom.tsx

```diff
@@ -18,6 +20,15 @@ const EmergencyViewRoom: React.FC<EmergencyViewRoomProps> = ({
   isLive,
 }) => {
   const { enterRoom, recentRoomUsers } = useRoom({ room, venueName });
+  const roomImage = useMemo(
```

Comment:
We don't actually need the useMemo here, but I know it's still a topic of debate :)

Reply:
Yes we don't since it's returning the string. Remove it, please